requirements.txt

```diff
@@ -10,3 +10,4 @@ capstone==3.0
 hexdump
 nose
 ./qiradb
+../bap/python
```

Comment:
add a script to pull the latest bap python into the qira directory, this path isn't always present.